src/overflowMenu/vendor/MenuItem.js

```diff
@@ -30,6 +30,10 @@ export type Props = {|
    * TestID used for testing purposes
    */
   testID?: string,
+  /**
+   * Whether the 'item' is destructive action. Destructive items are formatted differently on iOS.
+   */
+  destructive?: boolean
```

Comment:
is this here just to satisfy flow or not? I find the comment a little misleading, because ifaik the `destructive` prop does not influence `MenuItem` in any way, or does it?

Reply:
I believe it does:
![Screenshot 2020-07-29 at 19 49 42](https://user-images.githubusercontent.com/4304122/88834863-b8af7580-d1d4-11ea-8073-1d7d29831428.png)

Reply:
the screenshot comes from the `overflowMenuPressHandlerActionSheet`, but `src/overflowMenu/vendor/MenuItem.js` is used in `overflowMenuPressHandlerDropdownMenu`. Even if you look at https://github.com/vonovak/react-navigation-header-buttons/blob/364a327b1e56bb62c3c6c3d14d68eb4e69c234d9/src/overflowMenu/vendor/MenuItem.js you'll see `destructive` is not used in there